dspace-api/src/main/java/org/dspace/health/Core.java

```diff
@@ -0,0 +1,267 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ *
+ * by lindat-dev team
+ */
+package org.dspace.health;
+
+import org.apache.commons.io.FileUtils;
+import org.dspace.content.Community;
+import org.dspace.content.Item;
+import org.dspace.content.ItemIterator;
+import org.dspace.content.Metadatum;
+import org.dspace.core.Context;
+import org.dspace.storage.rdbms.DatabaseManager;
+import org.dspace.storage.rdbms.TableRow;
+import org.dspace.storage.rdbms.TableRowIterator;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Core {
+
+    // get info
+    //
+    public static String getCollectionSizesInfo() throws SQLException {
+        String ret = "";
+        List<TableRow> rows = sql(
+            "SELECT "
+                + "(SELECT text_value FROM metadatavalue "
+                + "WHERE metadata_field_id=64 AND resource_type_id=3 AND resource_id=col.collection_id) AS name, "
```

Comment:
Replace hard-coded constants. Use:
SELECT 
(SELECT text_value FROM metadatavalue 
WHERE metadata_field_id=(SELECT metadata_field_id FROM metadatafieldregistry WHERE element='title' AND qualifier IS NULL AND metadata_schema_id=(SELECT metadata_schema_id FROM metadataschemaregistry WHERE short_id='dc'))
AND resource_type_id=3 AND resource_id=col.collection_id) AS name, SUM(bit.size_bytes) AS sum 
FROM collection2item col, item2bundle item, bundle2bitstream bun, bitstream bit 
WHERE col.item_id=item.item_id AND item.bundle_id=bun.bundle_id AND bun.bitstream_id=bit.bitstream_id 
GROUP BY col.collection_id;